src/CAREUI/interactive/FiltersSlideover.tsx

```diff
@@ -35,18 +33,13 @@ export default function FiltersSlideover({
         <div className="flex items-center justify-between">
           <span className="text-lg font-bold">{t("filters")}</span>
           <div className="mr-2 flex items-center justify-end gap-1">
-            <ButtonV2
-              variant="danger"
-              ghost
-              onClick={onClear}
-              id="clear-filter"
-            >
+            <Button variant="warning" onClick={onClear} id="clear-filter">
```

Comment:
let's keep it as ghost 
```suggestion
            <Button variant="ghost" onClick={onClear} id="clear-filter">
```